tests/testsuite/build.rs

```diff
@@ -5966,32 +6200,46 @@ fn target_filters_workspace() {
 
     ws.cargo("build -v --example ex")
         .with_status(101)
-        .with_stderr(
-            "\
+        .with_stderr_data(str![[r#"
 [LOCKING] 2 packages to latest compatible versions
 [ERROR] no example target named `ex`
 
-<tab>Did you mean `ex1`?",
-        )
+	Did you mean `ex1`?
+
+"#]])
         .run();
 
     ws.cargo("build -v --example 'ex??'")
         .with_status(101)
-        .with_stderr(
-            "\
+        .with_stderr_data(str![[r#"
 [ERROR] no example target matches pattern `ex??`
 
-<tab>Did you mean `ex1`?",
-        )
+	Did you mean `ex1`?
```

Comment:
Should we add a `[TAB]` literal?

Reply:
I don't think so, and maybe we should remove tabs. They are not really widely used.

Reply:
clap also used to use them and they were removed.  I wonder how much of these tabs was for consistency